app/src/main/kotlin/com/simplemobiletools/calendar/pro/views/MonthView.kt

```diff
@@ -429,6 +433,11 @@ class MonthView(context: Context, attrs: AttributeSet, defStyle: Int) : View(con
         return event.startTS != event.endTS && Formatter.getDateTimeFromTS(event.endTS) == Formatter.getDateTimeFromTS(date.seconds()).withTimeAtStartOfDay()
     }
 
+    private fun findLastDay(event: Event) =
```

Comment:
this is hard to read, type out the return value like (event: Event) : Day {... return.... please